engine/src/main/java/io/camunda/zeebe/engine/EngineConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine;
+
+import java.time.Duration;
+
+public final class EngineConfiguration {
+
+  public static final int DEFAULT_MESSAGES_TTL_CHECKER_BATCH_LIMIT = 10;
```

Comment:
:thought_balloon: Since the configuration parameter is declared as experimental, I think, the default value should be set to `Integer.MAX_VALUE`. That way, the behavior would remain the same as to prior these changes. WDYT?